#! /bin/sh /usr/share/dpatch/dpatch-run
## 09_sshcrypto.dpatch by Steve Kemp <skx@debian.org>
##
## DP: Missing openssl  includes in sshcrypto.c. 
## DP: This patch was through diff.gz and now is implemented as a dpatch.

@DPATCH@
diff -urNad dsniff-2.4b1~/sshcrypto.c dsniff-2.4b1/sshcrypto.c
--- dsniff-2.4b1~/sshcrypto.c	2006-11-02 23:41:11.000000000 -0300
+++ dsniff-2.4b1/sshcrypto.c	2006-11-02 23:41:55.000000000 -0300
@@ -14,6 +14,8 @@
 
 #include <sys/types.h>
 #include <openssl/ssl.h>
+#include <openssl/blowfish.h>
+#include <openssl/des.h>
 
 #include <err.h>
 #include <stdio.h>
